packages/preferences/src/components/preferences-modal-tabs/style.scss

```
@@ -1,12 +1,15 @@
 $vertical-tabs-width: 160px;
 
 .preferences__tabs-tablist {
```

```suggestion
[class].preferences__tabs-tablist {
```
Is a strong enough specificity hack to obviate `!important` usage here but this is pretty nit-picky given there’s likely no one trying to override those rules. Mostly sharing because it’s a handy hack in general.

---

Yeah, in fact `.preferences__tabs-tablist.preferences__tabs-tablist` works. However, I chose this explicitly for two reasons: 1. it is more obvious that we're forcing an override and 2. the root issue is that the component styles should not have precedence in the first place, so this code's readability should not suffer from it. As a bonus, doing it that way would be fragile since it could stop working if the underlying styles (which are an implementation details) starts using a selector with higher specificity for some reason.

Thanks for chiming in :)